### PR TITLE
Product feature

### PR DESCRIPTION
**Pull request by xavizus**
Added post product feature with tests.

### CODE REVIEW

File: controllers/productController.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const productModel = require('../models/productModel')
+
+class ProductController {
+
+    async createProduct(req, res, next) {
+        try {
+            let result = await productModel.createProduct(req.body)
+
+            res.status(200).send({
```
Comment (RickardPedersen) [inline controllers/productController.js]: Could just send result

**Review by RickardPedersen — CHANGES_REQUESTED** [review]
Please do the following changes

File: controllers/productController.js (hunk 1 above)
Comment (xavizus) [reply controllers/productController.js]: This is now fixed

**Comment by RickardPedersen** [comment]
Looks good!

**Pull request merged**